lib/ecto/changeset.ex

```diff
@@ -1111,7 +1113,9 @@ defmodule Ecto.Changeset do
 
   ## Options
 
-    * `:required` - if the embed is a required field
+    * `:required` - if the embed is a required field. For embeds with cardinality
+      one, a non-nil value satisfies this validation. For embeds with many entries,
+      a non-empty list is satisfactory.
```

Comment:
I think `embeds_one` can sound better than "embeds of cardinality one", same for `embeds_many`.